packages/models-library/src/models_library/rest_ordering.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
+
+from pydantic import BaseModel, Field
+
+
+class OrderDirection(str, Enum):
+    ASC = "asc"
+    DESC = "desc"
+
+
+class OrderBy(BaseModel):
+    """inspired by Google AIP https://google.aip.dev/132#ordering"""
+
+    field: str
```

Comment:
MIONR: user `IDStr` or simply add constraints to avoid that the string is never empty and is bounded! (good security practice)